projects/packages/my-jetpack/_inc/state/selectors.js

```diff
@@ -143,6 +143,37 @@ const getStatsCounts = state => {
 	return state.statsCounts?.data;
 };
 
+const getUserStats = state => {
+	return state.userStats;
+};
```

Comment:
nitpick: can we change this to `getLifecycleStats`? Also is `userStats` in the state?

Reply:
Ah yeah, I've changed it last moment everywhere but not here, thanks!